.yamato/config/universal_lightning.metafile

```diff
@@ -0,0 +1,45 @@
+project:
```

Comment:
small detail but there is a typo in the name of this file! it should be `universal_lighting`